html/index.html

```diff
@@ -33,16 +33,55 @@
         }
         geoGetCurrentLocation();
 
-        $(document).ready(function(){
-            $(":submit").click(function(){
+        // Endpoint: http://localhost:3000/outlets?searchWord=kfc&currentLatitude=1.3730702&currentLongitude=103.9028103
+        // NOTE: code not working
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault <-- maybe you need to do this to prevent the form from posting.